### PR TITLE
Verify sidecar images creation during release

### PR DESCRIPTION
**Pull request by mkuznyetsov**
Signed-off-by: Mykhailo Kuznietsov <mkuznets@redhat.com>

### What does this PR do?
Add extra check to verify that sidecar images are released before devfile registry release

### What issues does this PR fix or reference?

https://github.com/eclipse/che/issues/19490

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [ ] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [ ] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [ ] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [ ] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [ ] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [ ] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [ ] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

**Review by nickboldt — APPROVED** [review]
LGTM ... but evidently I should have tried to RUN this :( 

Fixed in https://github.com/eclipse-che/che-devfile-registry/pull/375

File: make-release.sh [hunk 1]
```diff
@@ -27,6 +27,65 @@ usage ()
...
+}
+
+checkRequiredImagesExist()
+{
+  IMAGE_QUAY_PREFIX="quay.io/eclipse"
+  TAG=$(head -n 1 VERSION)
+  while read -r line; do
+    IMAGE_NAME=$(echo "$line" | tr -s ' ' | cut -f 1 -d ' ')
+    verifyContainerExistsWithTimeout "${IMAGE_QUAY_PREFIX}/${IMAGE_NAME}:${TAG}" 1
+  done < "${SCRIPT_DIR}"/base_images
```
Comment (Prabhav-Thali) [inline make-release.sh:86]: @mkuznyetsov looks like recent [build](https://github.com/eclipse-che/che-devfile-registry/actions/runs/770647804) has failed as variable `SCRIPT_DIR` is not set.

Comment (nickboldt) [reply make-release.sh:86]: Missing declaration is:

```
SCRIPT_DIR=$(cd "$(dirname "$0")" || exit; pwd)
```

Should have been reported by shellcheck plugin if you're using vscode?

Comment (nickboldt) [reply make-release.sh:86]: Fix in https://github.com/eclipse-che/che-devfile-registry/pull/375

**Pull request merged**